src/CMakeLists.txt

```diff
@@ -92,6 +92,13 @@ if(CELERITAS_USE_Geant4)
   add_subdirectory(accel)
 endif()
 
+if(CELERITAS_USE_VecGeom AND CELERITAS_USE_Geant4)
+  celeritas_target_include_directories(celeritas
+    PUBLIC
+    $<BUILD_INTERFACE:${Geant4_INCLUDE_DIRS}>
+  )
+endif()
+
```

Comment:
Dude I told you to remove this. The reason you were missing CLHEP directories is because you're only including Geant4 dirs here and you need to include CLHEP as well. Even if you did add this code, it should've been in `src/celeritas` rather than here, and because it was here I just wasted 20 minutes of my time trying to figure out why geant4 is being added to all the  celeritas source code when it shouldn't.